dbms/src/Flash/Mpp/newMPPExchangeWriter.h

```diff
@@ -32,6 +32,7 @@ std::unique_ptr<DAGResponseWriter> newMPPExchangeWriter(
     tipb::CompressionMode compression_mode,
     Int64 batch_send_min_limit_compression,
     const String & req_id,
-    bool is_async = false);
+    bool is_async = false,
+    bool selective_block = false);
```

Comment:
Does this meas a writer can only support `selective_block` or non-selective_block? If yes, it seems lack of flexibility

Reply:
done. will check `block.info.selective` in runtime.